### PR TITLE
TF-206 An Example Of Progress Buttons

### PR DESCRIPTION
**Pull request by PeterNancollis-UH**
- [x] Code Review 
- [x] Developer Testing: Key item tests 
- [ ] Product Owner Review (Optional)

### CODE REVIEW

File: Examples-Web/AnimatedProgressButtons/LinearProgressAnimation.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+function AddTouchFreeCursor()
+{
+    var cursorRing = document.createElement('img');
+    var cursor = document.createElement('img');
+    cursor.src = "./images/Dot.png";
+    cursor.style.position = "absolute";
+    cursor.width = 75;
+    cursor.height = 75;
+    cursor.style.zIndex = "1001";
+    // cursor.style.filter = ("drop-shadow(0px 0px 5px slategray");
```
Comment (MattGrayUL) [inline Examples-Web/AnimatedProgressButtons/LinearProgressAnimation.js]: Is this needed/useful?

**Review by MattGrayUL — COMMENTED** [review]
Just one small comment on the code itself.

I wonder if we should consider having 'Simple' and 'Extra Examples' folders in these examples. There's a lot of stuff in this that isn't _totally_ necessary but is useful to show varied use cases etc.

File: Examples-Web/AnimatedProgressButtons/LinearProgressAnimation.js (hunk 1 above)
Comment (PeterNancollis-UH) [reply Examples-Web/AnimatedProgressButtons/LinearProgressAnimation.js]: I forgot to remove that line, but have removed it now

**Review by MattGrayUL — APPROVED** [review]

**Comment by PeterNancollis-UH** [comment]
> I wonder if we should consider having 'Simple' and 'Extra Examples' folders in these examples. There's a lot of stuff in this that isn't totally necessary but is useful to show varied use cases etc.

In response to this, I am also unsure about how best to deal with the amount of non-essential code in the example.  I would like to see these examples going beyond bare-bones demonstrations so that they can serve more as inspiration and putting them into some simple interaction context can really help with that. 

I agree it would be good to maintain a clear separation of the example code and supporting code though so if we can find a better way to do that it would be great

**Pull request merged**